example-zowe.yaml

```diff
@@ -318,13 +318,14 @@ zowe:
     server:
       tls:
         attls: false
-        # TLS settings only apply when attls=false
-        # Else you must use AT-TLS configuration for TLS customization.
+        # If you have changed 'attls: true' above, the TLS and SSL levels are defined in the AT-TLS policy, 
+        # see https://docs.zowe.org/stable/user-guide/configuring-at-tls-for-zowe-server/
+        # Only include the two lines below that control the min and max TLS if 'attls: false' above
         minTls: "TLSv1.2"
         maxTls: "TLSv1.3"
     client:
```

Comment:
the easiest solution for 'client' is just to get rid of it from the example.
it's an advanced option, and, allowing attls to be toggled on just client was a mistake so i think our code doesnt even respect it now.